packages/app/src/app/pages/Dashboard/Content/routes/Recent/index.tsx

```diff
@@ -65,29 +68,37 @@ export const Recent = () => {
         .slice(0, 12);
 
   const pageType: PageTypes = 'recent';
+  const showUpgradeBanner = isFree && isTeamSpace;
   const isEmpty = !dataIsLoading && items.length === 0;
 
+  if (isEmpty) {
+    return <EmptyRecent showUpgradeBanner={showUpgradeBanner} />;
+  }
+
   return (
-    <SelectionProvider
-      activeTeamId={activeTeamId}
-      page={pageType}
-      items={items}
-    >
+    <SelectionProvider activeTeamId={activeTeam} page={pageType} items={items}>
       <Helmet>
         <title>Recent - CodeSandbox</title>
       </Helmet>
-      {isFree && isTeamSpace && <UpgradeBanner teamId={activeTeamId} />}
+      {showUpgradeBanner && (
+        <Element
+          css={{
+            width: `calc(100% - ${2 * GUTTER}px)`,
+            maxWidth: `calc(${GRID_MAX_WIDTH}px - 2 * ${GUTTER}px)`,
```

Comment:
why does the banner need both the maxWidth and exact width calculation? I know this was carried over but feels fishy :)

Reply:
This is needed to make the content fit the same space as the variable grid. Not sure why it wasn't implemented globally withing the sandbox content element.